cmake/BLTPrivateMacros.cmake

```diff
@@ -83,6 +83,34 @@ function(blt_error_if_target_exists target_name error_msg)
     endif()
 endfunction()
 
+##-----------------------------------------------------------------------------
+## blt_fix_fortran_openmp_flags()
+##
+## Fixes the openmp flags for a Fotran target if they are different from the
+## corresponding C/C++ openmp flags.
+##-----------------------------------------------------------------------------
+function(blt_fix_fortran_openmp_flags target_name)
+
+
+    if (ENABLE_FORTRAN AND ENABLE_OPENMP AND BLT_OPENMP_FLAGS_DIFFER)
+
+        get_target_property(target_link_flags ${target_name} LINK_FLAGS)
+        if ( target_link_flags )
+
+            message(STATUS "Fixing OpenMP Flags for target[${target_name}]")
```

Comment:
Will this message be too verbose for projects with a lot of fortran targets? Perhaps the message should be moved to ``SetupOpenMP``, where it will only be printed once (but will not list the targets that it modifies?

Reply:
Let's leave it for now. I'd rather see an explicit message that flags are being modified under the hood. We can change it later if it gets annoying.